### ISSUE TITLE
Sonar: 49 reliability issues

### ISSUE THREAD

**Issue by qmonmert** [jhipster/generator-jhipster#26106]
https://sonarcloud.io/project/issues?impactSoftwareQualities=RELIABILITY&resolved=false&id=jhipster-sample-application

<img width="468" alt="image" src="https://github.com/jhipster/generator-jhipster/assets/9989211/fe64c3e5-62b5-4525-976b-86f9de0ff701">

**Comment by mshima** [comment]
IMO 42 of 49 are false positives.
Doesn’t make sense to require keypress on aria-hidden elements that are not focusable.

**Comment by qmonmert** [comment]
@mshima ok I remove the `42` on the PR
